mongo/datadog_checks/mongo/mongo.py

```diff
@@ -218,7 +218,10 @@ def _check(self):
         if isinstance(deployment, ReplicaSetDeployment) and deployment.is_arbiter:
             dbnames = []
         else:
-            dbnames = api.list_database_names()
+            if self._config.db_names is None:
+                dbnames = api.list_database_names()
+            else:
+                dbnames = self._config.db_names
             self.gauge('mongodb.dbs', len(dbnames), tags=tags)
```

Comment:
We should not alter this metric, it is defined as
> Total number of existing databases

Reply:
What do you suggest? Add a new metric that is not the total but the configured ones?

Reply:
I suggest still fetching them all but not going over them collecting metrics from each one. I assume is not that one query the one that's overloading the cluster but rather the collecting that's happening later